frontend/src/bundles/common/components/badge/badge.tsx

```diff
@@ -0,0 +1,90 @@
+import { HeadphonesOutlined } from '@mui/icons-material';
+import { SvgIcon } from '@mui/material';
+import { type ReactElement } from 'react';
+
+import { BadgeColors } from '../../enums/badge-colors.enum.js';
+import { getValidClassNames } from '../../helpers/helpers.js';
+import styles from './styles.module.scss';
+
+type Properties = {
+    primaryText: string;
+    description: string;
+    secondText?: string;
+    color?: keyof typeof BadgeColors;
+    size?: keyof typeof Size;
+    icon?: ReactElement;
+};
+
+type IconProperties = {
+    icon: ReactElement | undefined;
+    iconClass?: string;
+};
+
+const Size = {
+    DEFAULT: 'DEFAULT',
+    SMALL: 'SMALL',
+};
+
+const Icon: React.FC<IconProperties> = ({ icon, iconClass }) => {
+    return icon ? (
+        <SvgIcon className={iconClass}>{icon}</SvgIcon>
+    ) : (
+        <HeadphonesOutlined className={iconClass} />
+    );
+};
+
+const Badge: React.FC<Properties> = ({
+    primaryText,
+    description,
+    secondText,
+    color = 'BLUE',
+    size = 'DEFAULT',
+    icon,
+}) => {
+    const isSmall = Size[size] === Size.SMALL;
+    const setClass = (classStandart: string, classSmall: string): string =>
+        getValidClassNames(classStandart, isSmall ? classSmall : '');
+
+    return (
+        <article
```

Comment:
we have Grid component for it

Reply:
I used mui Grid, because of our custom Grid component user prop childrento pass children, and eslint prevent me to pass in that way due to `react/no-children-prop`